compiler_setup/intel.sh

```diff
@@ -2,7 +2,7 @@
 F90=ifort
```

Comment:
I guess the various `compiler_setup/*.sh` files should have copyright/licence notices too.